gulpfile.js

```diff
@@ -0,0 +1,214 @@
+const gulp = require('gulp');
+const replace = require('gulp-replace');
+const rename = require('gulp-rename');
+const logger = require('gulplog');
+const path = require('path');
+const fs = require('mz/fs');
```

Comment:
what's wrong with the builtin `fs` package?

Reply:
Code from polymer-cli